docs/source/releases.md

```diff
@@ -60,14 +60,14 @@ GA images aren't build from scratch but rather promoted from an existing release
 Support matrix table shows the version requirements for a particular **scylla-operator** version. Be sure to match these requirements, otherwise some functionality will not work.
 
 :::{table}
-| Component         | v1.13      | v1.12      | v1.11      | v1.10      |
-|:-----------------:|:----------:|:----------:|:----------:|:----------:|
-| Kubernetes        | `>=1.21`   | `>=1.21`   | `>=1.21`   | `>=1.21`   |
-| CRI API           | `v1`       | `v1`       | `v1`       | `v1`       |
-| Scylla OS         | `>=5.4`    | `>=5.0`    | `>=5.0`    | `>=5.0`    |
-| Scylla Enterprise | `>=2023.1` | `>=2021.1` | `>=2021.1` | `>=2021.1` |
-| Scylla Manager    | `>=3.2.8`  | `>=3.2.6`  | `>=3.2`    | `>=2.6`    |
-| Scylla Monitoring | `(CRD)`    | `(CRD)`    | `(CRD)`    | `>=4.0`    |
+| Component         | master                 | v1.14                  | v1.13                  |
```

Comment:
won't adding master make us have to update the table whenever there's a new release of any of the products in rows?

Reply:
I don't see how it differs from our other versions.

The intent of having the master is so that we can raise the requirement together with the feature that needs it and it's not forgotten when doing the release a month or so later.

Reply:
> I don't see how it differs from our other versions.

We don't bump the support matrix of already released versions.

Say a new ScyllaDB OS version comes out - when is the upper limit for master going to be bumped? Only when we bump the versions we test? That only happens when someone spontaneously does so, so in practice this may end up saying we don't support a current release of ScyllaDB with master just because nobody bumped the docs?

Reply:
This is exactly what's being asked of us.

Reply:
ok